include/aikido/constraint/uniform/detail/RnConstantSampler-impl.hpp

```diff
@@ -1,11 +1,13 @@
-#include <aikido/constraint/uniform/RnConstantSampler.hpp>
-
 #include <stdexcept>
-
 #include <dart/dart.hpp>
+#include "aikido/constraint/uniform/RnConstantSampler.hpp"
+
+#undef dtwarn
+#define dtwarn (::dart::common::colorErr("Warning", __FILE__, __LINE__, 33))
```

Comment:
Is this because of namespace `dart` ambiguous between `::dart` and `::aikido::[~]::dart`? Let me make an upstream fix for this.

Reply:
Yep. I created #348 to track this.